lib/net.c

```diff
@@ -552,6 +606,10 @@ gatekeeper_init_network(struct net_config *net_conf)
 	RTE_ASSERT(net_conf->back->num_rx_queues <= GATEKEEPER_MAX_QUEUES);
 	RTE_ASSERT(net_conf->back->num_tx_queues <= GATEKEEPER_MAX_QUEUES);
 
+	/* Convert RSS key. */
+	rte_convert_rss_key((uint32_t *)&default_rss_key,
```

Comment:
As the ``rte_convert_rss_key()`` is only called once in ``gatekeeper_init_network()``, instead of called by ``init_iface()``, so I think the current position is fine?